docs/ssdt/release-notes-ssdt.md

```diff
@@ -40,6 +40,35 @@ GeneMi , 2019/03/22.
 P.S.  there's no need to keep this large HTML comment indefinitely.
 -->
 
+## 15.9.6,&nbsp; SSDT for VS 2017
+
+_Released:_ &nbsp; Oct 27, 2020  
```

Comment:
Let's fix this date to one not in the future

Reply:
Nice catch for the typo error. Will update it.

Reply:
Created new PR #5389 and will close this one.